docs/Plugin Architecture/Plugin Management.md

```diff
@@ -155,12 +155,6 @@ cli plugin list
 
 No arguments are sent to this command as it is meant to list all installed plugins and their status. That is good enough for now and could be built upon later.
```

Comment:
I know that there are many things wrong in this document that we do not want to undertake at this time. So feel free to reject this suggestion. However, it might be simple because it is mostly deletions. 

I do not think that any of the following plugin commands exist: health, version, configure, help, repair, add, or clean.
I think that we should just delete them.

For the remaining examples, relace 'cli plugin' with 'zowe plugins'.

Reply:
I removed those, and all references to config, as it was for the old config commands.

Reply:
LGTM